test3.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * Model file to handle all repository actions
+ */
+const { repoSchema, tempRepoSchema } = require('../schema/index');
+const { Logger, DB } = require('../helpers/index');
+const path = require('path');
+
+/**
+ * Get Repo model from ORM
+ *
+ * @async
+ * @function GetRepoModel
+ * @params {string} - org
+ * @params {boolean} - schedulerEnabled
+ * @returns {object}
+ * @author dev-team
+ */
+
+const GetRepoModel = async (org = null, schedulerEnabled = true) => {
+  try {
+    const model = schedulerEnabled ? { name: 'Repo', schema: repoSchema } : { name: 'TempRepo', schema: tempRepoSchema };
```

Comment:
Codacy has a fix for the issue: [Replace `·?·{·name:·'Repo',·schema:·repoSchema·}·:·{·name:·'TempRepo'` with `⏎······?·{·name:·"Repo",·schema:·repoSchema·}⏎······:·{·name:·"TempRepo"`](https://app.codacy.com/gh/thamizhMsys/testRepo6/pullRequest?prid=9650715)
```suggestion
    const model = schedulerEnabled
```